tests/dispersytestclass.py

```diff
@@ -18,7 +25,7 @@
 # use logger.conf if it exists
 if os.path.exists("logger.conf"):
     # will raise an exception when logger.conf is malformed
-    logging.config.fileConfig("logger.conf")
+    logging.basicConfig(filename="logger.conf")
```

Comment:
This code was normally not executed during tests for dispersy, because a logger.conf is not available by default in Dispersy.
But I started testing from Tribler where one is available by default. This resulted in an exception, because the logging.config is not imported correctly.

If config.fileConfig is used, then during test execution all debug statements are logged. While if basicConfig is used, the tests have clean logging statements.